lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
+    &:not(.nx-list__item--empty):not(.disabled):not([disabled]),
+    &:not(.nx-list__item--error) {
       &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
+        background-color: #f1f3fa;
         cursor: pointer;
         outline-style: none; // removes :focus outline
 
         .fa-chevron-right {
-          color: $nx-color-blue-41;
+          color: #045dd1;
         }
       }
 
       &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+        background-color: #d3e6ff;
       }
     }
 
     &.nx-list__item--empty {
       @include regular();
 
-      border-top: $nx-border; // assume it's always the first item
       color: $nx-text-color-light;
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
-
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
-
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
-      }
-    }
-
     .nx-list__subtext {
       pointer-events: none; // Fix for IE11 :active state display
```

Comment:
Is this still necessary?

Reply:
> Is this still necessary?

I can't figure out why it's there, it's a holdover from something for IQ, removed: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/8e5d1365db4c91fd78145c309fc151e35f771c4a